### PR TITLE
Update External_data_sources.md

### PR DESCRIPTION
**Pull request by r2sunita**
Updated  with additional external source information on Ensembl Plants/Gramene and Phytozome. Thanks. 
Sunita

### CODE REVIEW

File: External_data_sources.md [hunk 1]
```diff
@@ -178,3 +178,39 @@ Creative Commons Attribution-NoDerivs License
...
+---
+
+### Phytozome
+
+#### How Phytozome is used in KBase
+
+Phytozome is the public portal to JGI's plant genome data and analysis. Currently KBase uses Phytozome as an additional resource for released plant genomes which are not available at Ensembl Plants/Gramene. Phytozome will also be used to provide diversity, expression data and many other data types to KBase in future. 
+
+#### License for Phytozome
+Everything is open source.
```
Comment (fperez) [inline External_data_sources.md]: Could you provide here a specific description of which license it uses? Something like "The code is available under the terms of the BSD license, full details at http://foo.bar.com/license" or somesuch...

**Comment by fperez** [comment]
Thanks @r2sunita! Very small comment, otherwise good to merge once this small addition is made.

**Comment by r2sunita** [comment]
Yes. I have contacted Phytozome group to get the specific details on license. I will provide the details as soon as I get this information. Thanks. Sunita

**Comment by fperez** [comment]
Excellent, thanks!

**Comment by r2sunita** [comment]
Hi Fernando, 

I contacted Phytozome lead Dr. David Goodstein to get the specific details on license. Here is his reply. 
1. We don’t license or distribute any code associated with Phytozome
2. Phytozome is ©2006-2015 University of California Regents. All rights reserved.
3. go to http://genome.jgi.doe.gov/pages/dynamicOrganismDownload.jsf?organism=PhytozomeV10 (the data download Portal for Phytozome) and view the header paragraph:

Some of the data available on this portal are pre-publication data and may contain errors. The goal of our policy is that early release should enable the progress of science. By accessing these data, you agree not to publish any articles containing analyses of genes or genomic data on a whole genome or chromosome scale prior to publication by JGI and its collaborators of its comprehensive genome analysis. These restrictions will be lifted on the publication of the whole genome description or as specified on the information for each organism. During this waiting period, the data will be available for any kind of publication that does not compete directly with planned publications (e.g. reserved analyses) of the JGI and collaborators. A principal collaborator or "champion," listed in the organsim's Info page and is the point of contact and arbiter regarding publication plans. Scientists are strongly encouraged to contact the principal collaborator and JGI about their intentions and any potential collaboration.

Each directory within this portal has a file describing the release policy of that data. People who access data are expected to adhere to those policies.

**Comment by r2sunita** [comment]
I have updated the "External_data_sources.md"  with Phytozome data usage policy. Thanks. Sunita

**Comment by fperez** [comment]
Great, thanks Sunita! Merging now.

**Pull request merged**